packages/Sortable/src/components/SortableItem/SortableItem.styles.js

```diff
@@ -0,0 +1,66 @@
+import { css } from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const draggingStyles = `
+  background: ${tokens.color.white};
+  box-shadow: ${tokens.shadow}, 0 0 1px ${tokens.border.color};
+
+  &:focus {
+    box-shadow: ${tokens.shadow}, ${tokens.highlight.active.withBorder.boxShadow};
+  }
+`;
+
+export const itemStyles = css`
+  border-radius: ${tokens.border.radius};
+  display: flex;
+  list-style: none;
+  padding: ${tokens.space} ${tokens.spaceSm} ${tokens.space} 2px;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  &:focus {
+    outline: none;
+    box-shadow: ${tokens.highlight.active.withBorder.boxShadow};
+  }
+
+  ${({ isDragging }) => (isDragging ? draggingStyles : null)}
+`;
+
+export const itemHandleStyles = `
```

Comment:
Q: are we only using the `css` helper when there are functions inside?

Reply:
css let you receive the props inside the function but also helps IDEs/linters to highlight css and prettier fix issues. 

I guess we should use css in all cases unless someone finds a significant performance issue by using the css method, in that case, using the string assignment should be fine.